web/controllers/admin_controller.ex

```diff
@@ -319,6 +319,7 @@ defmodule ExAdmin.AdminController do
   end
 
   defp to_integer(:string, string), do: string
+  defp to_integer(:id, string), do: string
```

Comment:
Hello Steve, 

I am having difficulty setting things up for test.
I see that content of config/dev.exs  is empty Do we have to set this up as the umbrella project. 
I am checking umbrella project to know how to. If you have short cut way or any notes please let me know.